docker-compose.override.yml

```diff
@@ -67,3 +67,8 @@ services:
       - LICENSE_KEY
     volumes:
       - ./../mint-infra-modules/lambda/prince/build:/var/task:ro,delegated
+  email:
+    image: dockage/mailcatcher:0.7.1
+    ports:
+      - "1085:1080"
+      - "1030:1025"
```

Comment:
Why is this being added when it already exists in `docker-compose.yml`? Seems like this is duplicative of what's in that file.

Reply:
Good point. Leftovers from debugging the mailcatcher connectivity issues. Removed,